examples/data-objects/prosemirror/src/index.ts

```diff
@@ -15,9 +15,12 @@ import {
 	RuntimeFactoryHelper,
 } from "@fluidframework/runtime-utils";
 import { MountableView } from "@fluidframework/view-adapters";
-import { ProseMirror, ProseMirrorFactory, ProseMirrorView } from "./prosemirror";
 
-export { ProseMirror, ProseMirrorFactory, ProseMirrorView } from "./prosemirror";
+import React from "react";
+
+import { ProseMirror, ProseMirrorFactory, ProseMirrorReactView } from "./prosemirror";
+
+export { ProseMirror, ProseMirrorFactory, ProseMirrorReactView } from "./prosemirror";
```

Comment:
Spacing, plus looks like dupe imports?

Reply:
Prettier doesn't care - I can remove the newline if you like.

Some of the older demos like this one do a weird import/export thing which is what's happening here.

Reply:
I completely missed that one was export. Carry on.